dev-tools/composer.json

```diff
@@ -9,7 +9,8 @@
         "phpstan/extension-installer": "^1.3.1",
         "phpstan/phpstan": "^1.10.51",
         "phpstan/phpstan-phpunit": "^1.3.15",
-        "phpstan/phpstan-strict-rules": "^1.5.2"
+        "phpstan/phpstan-strict-rules": "^1.5.2",
+        "symfony/var-dumper": "^6.4 || ^7.0"
```

Comment:
no opinion on my side as I'm not using this.

but wouldn't it be easier if we would include it in master composer.json file?

Reply:
> Adding `require __DIR__.'/dev-tools/autoload.php';` to `.php-cs-fixer.php` can help […]

This would also remove the need to do this as the autoloader will already include the functions if I remember correctly.

Reply:
Yeah, I believe @keradus considered "easier" exactly because of that 😉. I've moved requirement.